### DISCUSSION TITLE
Change O2M objects order with drag and drop

### DISCUSSION THREAD

**Discussion by mhdaouas** [directus/directus#8462] [Archive]
I have a single type object which has an O2M field. It seems impossible to re-order O2M objects inside it. In my case the objects are Directus roles and they are ordered by role id by default (x-x-x-x-x). It would be nice to be able to re-order them with drag and drop. I want to use this to configure workflow steps based on validation roles. What would be a good workaround in case this feature is not implemented?

**Comment by rijkvanzanten** [comment]
If you go into the relationship setup for this particular o2m field, there's a _Sort Field_ option you can configure. This will enable manual ordering 👍🏻

**Reply by mhdaouas** [reply]
Wow you've thought about everything! Bravo!
